docs/kyma-environment-broker/04-02-cls-account-setup.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Cloud Logging Service Account setup
+type: Details
+---
+
+The image below explains the account setup for SKR and CLS instances
+
+![CLS Account diagram](./assets/cls-acc.jpeg)
+
+
+The CLS instances are created inside a sub account inside the Kyma Global Account. There is one CLS instance created for every customer SKR global account. This CLS instance is created for a given customer Global account when the customer provision first SKR instance.
+
+From the diagram above we can see when customer with Global Account X provisions first SKR (SKR 1) this would provsion a CLS instance (CLS for customer X) in the CLS sub account which is part of kyma global account. After provision of CLS instance, a binding is created and the credentials are passed over to SRK 1 so that the logs could be pushed to the CLS instance.
```

Comment:
```suggestion
The diagram illustrates the following: 
1. When a customer with global account X provisions the first SKR (SKR 1), a CLS instance (CLS for customer X) is created in the CLS subaccount that is part of the Kyma global account. 
2. After provisioning a CLS instance, a binding is created, and the credentials are passed over to SRK 1.  
3. With this setup, the logs are pushed to the CLS instance.
```

If this setup is not enough to push the logs (and other steps elsewhere are needed), we should adjust the third sentence here. Pls advise